.circleci/config.yml

```diff
@@ -0,0 +1,83 @@
+version: 2.1
+executors:
+  docker-publisher:
+    environment:
+      IMAGE_NAME: dsier/exerclist
+    docker:
+      - image: circleci/buildpack-deps:stretch
+jobs:
+  build:
+    docker:
+      - image: circleci/php:7.2
+
+    steps:
+      - checkout
+
+      - run: sudo apt update
+      - run: sudo apt install make
+      - run: sudo docker-php-ext-install zip
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            # "composer.lock" can be used if it is committed to the repo
+            - v1-dependencies-{{ checksum "composer.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run: composer install -n --prefer-dist
+
+      - save_cache:
+          key: v1-dependencies-{{ checksum "composer.json" }}
+          paths:
+            - ./vendor
+
+      - run: make test
```

Comment:
@daria-sieroshtan
this command currently is run outside, so it checks that code works with the host
Please change so it checks that code works with the container